### ISSUE TITLE
play-1.js: add support for speedup ratio

### ISSUE THREAD

**Issue by joaocc** [moinejf/abc2svg#22]
This one is quite simple to explain.
We would like to be able to speed up / slow down the playback by a factor of N. This could be made available to users via a slider.

After reading through play-1.js, it seems that the only way to do it now is to redo the abc conversion with a different BPM setting.
Something like "set_speed_factor" with a float (<1 == slowdown, >1 == speedup) would be great.

Thanks

**Comment by joaocc** [comment]
Added an initial very rough proof-of-concept (in the e29a47db97ea3fe94662cec135a1cc513d8c0acb commit on our fork)

**Comment by moinejf** [comment]
Your commit should be in the issue #21.

**Comment by joaocc** [comment]
You are correct. Sorry for the mixup. Will remove the comment to prevent misleading link.

**Comment by joaocc** [comment]
Not sure if this is the most elegant way to implement this, but we tried to keep changes to a minimum.

**Comment by moinejf** [comment]
I already implemented such a function in a previous player and I was about to code it.
(your proof of concept does not work because the realtime origin must be shifted)
